tests/Unit/RuleSet/DeclarationBlockTest.php

```diff
@@ -5,14 +5,20 @@
 namespace Sabberworm\CSS\Tests\Unit\RuleSet;
 
 use PHPUnit\Framework\TestCase;
+use Sabberworm\CSS\CSSElement;
 use Sabberworm\CSS\CSSList\CSSListItem;
+use Sabberworm\CSS\Position\Positionable;
+use Sabberworm\CSS\Rule\Rule;
 use Sabberworm\CSS\RuleSet\DeclarationBlock;
+use Sabberworm\CSS\RuleSet\RuleSet;
 
 /**
  * @covers \Sabberworm\CSS\RuleSet\DeclarationBlock
  */
 final class DeclarationBlockTest extends TestCase
```

Comment:
Are any of the changes to this testcase already possible now, i.e., could we move them to a pre-patch? (Haven't checked in detail yet, though.)

Reply:
The `assertInstanceOf` tests can be added separately - #1295 for this.